### PR TITLE
Add learn caption for Meirzv and Gian

### PR DESCRIPTION
**Pull request by Meirzv**

### CODE REVIEW

**Comment by fahadun** [comment]
its not working.

**Review by fahadun — APPROVED** [review]
emoji ? 👍

File: _posts/0000-01-08-Meirzv.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Everyone is cool ;]
```
Comment (kamanngpseg) [inline _posts/0000-01-08-Meirzv.md:6]: I agree!!!!

File: _posts/0000-01-04-GianfrancoLeto.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Upvote Meirzv
```
Comment (kamanngpseg) [inline _posts/0000-01-04-GianfrancoLeto.md:6]: 🥇

**Review by kamanngpseg — CHANGES_REQUESTED** [review]
Request for MORE CAPTION !!! 🔢

**Review by kamanngpseg — APPROVED** [review]
Aight!!! you got it!!

**Review by kamanngpseg — APPROVED** [review]
I approved

**Pull request merged**